src/katsdpcontroller/generator.py

```diff
@@ -1104,7 +1104,15 @@ def get_baseline_index(a1, a2):
                 n_accs=n_accs,
             )
         elif isinstance(stream, product_config.GpucbfTiedArrayChannelisedVoltageStream):
-            # TODO: NGC-1225
+            telstate_data.update(
+                # Had to keep mismatch in spelling as external users still use
+                # 'cent-e-r' spelling.
+                center_freq=stream.centre_frequency,
+                n_chans=stream.n_chans,
+                scale_factor_timestamp=stream.adc_sample_rate,
+                sync_time=sync_time,
+                ticks_between_spectra=stream.n_samples_between_spectra,
+            )
```

Comment:
See [this comment](https://skaafrica.atlassian.net/browse/NGC-1225?focusedCommentId=223640): we don't need any of these (SDP gets them from the upstream antenna-channelised-voltage stream), but we do need spectra_per_heap.

Reply:
My apologies once more. Especially because spectra_per_heap is the only one I left out 🤦